python/ct/client/log_client.py

```diff
@@ -232,6 +236,22 @@ def check_response_status(code, reason, content='', headers=''):
         else:
             raise HTTPError("%s (%s) %s" % (reason, content, headers))
 
+    @staticmethod
+    def _uri_with_params(uri, params=None):
+        if not params:
+            return uri
+        components = list(urlparse.urlparse(uri))
+        if params:
+            # Update the URI query, which is at index 4 of the tuple.
+            components[4] = urllib.urlencode(params)
+        return urlparse.urlunparse(components)
+
+    @staticmethod
+    def _build_requests_style_response((resp_hdr, resp_body)):
+        status_code = int(resp_hdr.pop("status")) if "status" in resp_hdr else 0
+        reason = resp_hdr["reason"] if "reason" in resp_hdr else ""
```

Comment:
Please find attached the letter 'o', 'n', 's', 'e' and 'a' for complete and consistent naming of the response header and response body parameters.